### PR TITLE
Random improvements

### PR DESCRIPTION
**Pull request by don-smith**
@Alroniks, thank you so much for putting this repo together and sharing it. It works perfectly!

As I was setting up my environment, I noticed some opportunities to tighten up the language in the README and thought I would share in case you might be interested in incorporating them.

I also replaced the use of `replace` in `init.sh` with `sed` since `sed` is everywhere and most people won't have `msql2mysql` installed.

Thanks again!

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   tao:
-    image: alroniks/tao
+    image: locksmithdon/tao
```
Comment (alroniks) [inline docker-compose.yml]: Why here another image? Could you show changes in Dockerfile?

**Comment by alroniks** [comment]
Hi @don-smith 
Thank you a lot for your impact! Sorry, I was in business trip and after on vacations so only now I can check it. And all seems nice except one question about docker image.

**Comment by alroniks** [comment]
I fixed image name and merged but the question is still actual.

File: docker-compose.yml (hunk 1 above)
Comment (don-smith) [reply docker-compose.yml]: Oh, I just wasn't sure if you were going to accept this PR so I updated the image name in a subsequent commit. I wouldn't expect you to accept that change. Sorry, I should have done this in a different branch and not my `master`. Thanks for accepting the PR and changing the Docker image back to yours.

**Pull request merged**